.env.example

```diff
@@ -0,0 +1,3 @@
+AIRTABLE_API_KEY=
+AIRTABLE_BASE_ID=
+AIRTABLE_TABLE_NAME=
```

Comment:
Nit pick here, but if you decide to keep the table name hard coded in the `getComponentDetails` function you may want to remove this variable from the example file.